dora/core/server/master/src/main/java/alluxio/master/job/UfsFileIterable.java

```diff
@@ -140,7 +140,7 @@ private FileInfo transformUfsStatus(UfsStatus ufsStatus) {
       AlluxioURI ufsUri = new AlluxioURI(PathUtils.concatPath(mPath,
           CommonUtils.stripPrefixIfPresent(ufsStatus.getName(), mPath)));
       FileInfo info = new FileInfo().setName(ufsUri.getName()).setPath(ufsUri.getPath())
-                                    .setUfsPath(ufsUri.getPath())
+                                    .setUfsPath(ufsUri.toString())
                                     .setFolder(ufsStatus.isDirectory())
                                     .setOwner(ufsStatus.getOwner()).setGroup(ufsStatus.getGroup())
```

Comment:
unrelated to this section, but in listFileInfos are we by default doing mFs.listStatus without doing recursive listing? listOptions.defaults doesnt set recursive is it?

Reply:
good catch, fixed